samples/legacy_support/All_ID_Mapping_Original.csv

```diff
@@ -1555,110 +1555,110 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 1553,Installation : Raw Material,S-G-IR----,S-G-IR----,,,,,Point,,,
 1554,Installation : Mine,S-G-IRM---,20111300,,,,,Point,,,
 1555,Installation : Missile and Space System Production,S-G-IMM---,20111400,,,,,Point,,,
-1556,Installation : Nuclear (Non CBRN Defense),S-G-IMF---,20111500,,,,,Point,,,
-1557,Installation : Nuclear (Non CBRN Defense) : No Category Listed : Atomic Energy Reactor,S-G-IMFA--,20111500,,20052,,,Point,,,
-1558,Installation : Nuclear (Non CBRN Defense) : No Category Listed : Nuclear Material Production,S-G-IMFP--,20111500,,20062,,,Point,,,
-1559,Installation : Nuclear (Non CBRN Defense) : No Category Listed : Nuclear Material Storage,S-G-IMFS--,20111500,,20072,,,Point,,,
+1556,Installation : Nuclear (Non CBRN Defense),S-G-IMF---,S-G-IMF---,,,,,Point,,,
+1557,Installation : Nuclear (Non CBRN Defense) : No Category Listed : Atomic Energy Reactor,S-G-IMFA--,S-G-IMF---,,20052,,,Point,,,
```

Comment:
@csmoore : this comment also applies to the next two lines.  I don't recommend mixing a new SVG file and 2525D modifier symbols.  JMSML will not produce this output.  My suggestion is to create three new SVG files that each incorporate the S-G-IMF--- base icon and the 2525D modifier graphics.